quarterback-packages/api/src/routes.ts

```diff
@@ -17,15 +17,14 @@ import { Router } from 'express'
 
 import { authenticate, validateBody } from '$middlewares'
 
-import * as authCtrl from './routes/auth/auth.ctrl'
+import * as statsCtrl from '$routes/stats/stats.ctrl'
 import * as commentCtrl from './routes/comment/comment.ctrl'
 import * as docCtrl from './routes/doc/doc.ctrl'
 import * as snapCtrl from './routes/snapshot/snap.ctrl'
 
 const router = Router()
 
-router.post('/authenticate', authCtrl.authenticate)
-router.get('/stats', authCtrl.stats)
+router.get('/stats', statsCtrl.stats)
```

Comment:
I wasn't thinking of removing it since I was trying to minimize the change set, but yeah it makes perfect sense to be removed since all it offers is an uptime